src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,22 @@
-import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies?: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+
+  <div className="movies">
+    {movies.map(movie => (
+      <div
+        key={movie.imdbId}
+        className="card"
+        data-cy="Movie"
+      >
```

Comment:
Move this wrapper into your MovieCard component. Because it belongs to the component
```
<div
   key={movie.imdbId}
   className="card"
   data-cy="Movie"
>
```

Reply:
Keep in mind that `key` shouldn't move inside MovieCard , it has to be passed like a prop
 `<MovieCard key={} {...movie} />`